balancer/endpointsharding/endpointsharding.go

```diff
@@ -124,16 +149,21 @@ func (es *endpointSharding) UpdateClientConnState(state balancer.ClientConnState
 		var bal *balancerWrapper
```

Comment:
Could you rename this local to `childBalancerWrapper` or something more clear -- at least `child` or something?  `bal` to me makes me think this is the `endpointsharding` balancer itself when I see it later on.  I guess we use `es` for that, but the more common convention is that the `Balancer` implementation within the package is simply named `balancer`, and `b` is the receiver.